WORKSPACE

```diff
@@ -75,19 +81,13 @@ http_archive(
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "142dd33e38b563605f0d20e89d9ef9eda0fc3cb539a14be1bdb1350de2eda659",
+    sha256 = "c1a5614131bbd4e4764ad782de261a2e9c4173c9528a50dbaefcbea6fce7d66a",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.22.2/rules_go-v0.22.2.tar.gz",
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.22.2/rules_go-v0.22.2.tar.gz",
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.21.7/rules_go-v0.21.7.tar.gz",
```

Comment:
why did you decide to downgrade io_bazel_rules_go . i think it was upgraded to support go 1.14

Reply:
Oh no! I didn’t mean to downgrade. Nice catch, i’ll Fix tomorrow

Reply:
Fixed